### PR TITLE
React UI: Added shortcuts

### LINKED ISSUE #948: Integrate the patch #941 to cvat-canvas library

### PR DESCRIPTION
**Pull request by bsekachev**
- Used: https://www.npmjs.com/package/react-hotkeys
- Press F1 to view all available shortcuts on the page
- 44 shortcuts have been added
- Some shortcuts haven't been moved from old UI because I am not sure they are applicable in the new one:
  - change_shape_label
  - change_shape_color
  - change_default_label
  - next_shape_type
  - prev_shape_type

TODO: In the future PRs: 
- Move all shortcuts to shortcuts state
- Add ability to change shortcuts from the UI
- Add UI elements to perform search (now shortcuts only)

Resolved #948

### CODE REVIEW

**Comment by ActiveChooN** [comment]
@bsekachev, It can be useful to add some other option to close help popup, except F1 button pressing. Something like cross or click away.

**Comment by ActiveChooN** [comment]
@bsekachev PASTE action works not so well. Redux fails after clicking `Ctrl+V` when other pasting action is active. I guess it happens due to some reducer still active.

**Comment by bsekachev** [comment]
@ActiveChooN 

Fixed

File: cvat-ui/src/reducers/annotation-reducer.ts [hunk 1]
```diff
@@ -419,6 +422,43 @@ export default (state = defaultState, action: AnyAction): AnnotationState => {
...
+            }
+
+            const { instance: canvasInstance } = state.canvas;
+            canvasInstance.draw({
+                enabled: true,
+                rectDrawingMethod: activeRectDrawingMethod,
+                numberOfPoints: activeNumOfPoints,
+                shapeType: activeShapeType,
+                crosshair: activeShapeType === ShapeType.RECTANGLE,
+            });
```
[the remark this answers is not in the document]
Comment (bsekachev) [reply cvat-ui/src/reducers/annotation-reducer.ts]: I thought about it. 
On the one hand we can consider this as changing of the state of another, independent storage (actually it is). On the other hand, I agree, it looks strange

**Pull request merged**